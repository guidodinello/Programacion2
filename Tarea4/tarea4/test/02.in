# Implementar enAlguno
enAlguno 0
enAlguno 1 4
enAlguno 2 4 8
enAlguno 5 1 3 7 8 9
reiniciar
agregarAIterador 2
enAlguno 0
enAlguno 1 4
enAlguno 2 4 8
enAlguno 5 1 3 7 8 9
enAlguno 1 2
reiniciar
agregarAIterador 4
agregarAIterador 7
agregarAIterador 9
enAlguno 0
enAlguno 1 4
enAlguno 2 4 8
enAlguno 5 1 3 7 8 9
enAlguno 6 1 3 7 8 9 11
Fin
