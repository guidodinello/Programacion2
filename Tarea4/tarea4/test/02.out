1>#  Implementar enAlguno.
2>
3>4 
4>4 8 
5>1 3 7 8 9 
6>Estructuras reiniciadas.
7>Agregando.
8>2 
9>2 4 
10>2 4 8 
11>1 2 3 7 8 9 
12>2 
13>Estructuras reiniciadas.
14>Agregando.
15>Agregando.
16>Agregando.
17>4 7 9 
18>4 7 9 
19>4 7 8 9 
20>1 3 4 7 8 9 
21>1 3 4 7 8 9 11 
22>Fin.
